docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -200,7 +203,11 @@ question: |
 subquestion: |
   If you can't finish now or need to get more information, you can save your work at any time. This lets you return later and finish, or go back and change any of your answers.
 
-  Go to the **menu at the top of the screen** and click "Sign in or sign up to save answers".
+  Below the **Continue** button, tap "Sign in" if you have an account or "Sign up" to make one. You can also go to the
+  % if device().is_mobile:
```

Comment:
Cool use of this! Per previous conversations, I gather that this isn't 100% reliable, but will catch it most of the time?

Reply:
Yep! It'll still just say "menu", no icon when people just have a narrow window open on desktop (< 800 pixels I think), and it'll show the icon when the actual menu doesn't have the icon when you have a really wide phone (think iPhone 11 in landscape).

But both of those cases are still understandable (one is our current text, and the other is adds one unnecessary icon, but isn't too distracting).